perun-auditlogger/src/main/java/cz/metacentrum/perun/auditlogger/service/impl/AuditLoggerManagerImpl.java

```diff
@@ -0,0 +1,96 @@
+package cz.metacentrum.perun.auditlogger.service.impl;
+
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import cz.metacentrum.perun.auditlogger.logger.EventLogger;
+import cz.metacentrum.perun.auditlogger.service.AuditLoggerManager;
+import cz.metacentrum.perun.core.api.Perun;
+import cz.metacentrum.perun.core.api.PerunClient;
+import cz.metacentrum.perun.core.api.PerunPrincipal;
+import cz.metacentrum.perun.core.api.PerunSession;
+
+@org.springframework.stereotype.Service(value = "auditLoggerManager")
+public class AuditLoggerManagerImpl implements AuditLoggerManager {
+
+	private final static Logger log = LoggerFactory.getLogger(AuditLoggerManagerImpl.class);
+
+	private final static String DEFAULT_CONSUMER_NAME = "auditlogger";
+	private final static String DEFAULT_STATE_FILE = "/tmp/auditlogger.state";
```

Comment:
I think we want this to be persistent, not disposable on server restart. Yet I'm not sure, how is this handled in dockerized version of LDAPc @martin-kuba ? On instances without docker, we have this property set to ``./[filename]`` which leads to the same folder as jar.
So there are two solutions. We can have default fallback value here and do not set property on each instance or we can explicitly set default in properties file and keep current default value (or any other required by the docker).

Reply:
I would argue that the default value should lead to somewhere guaranteed to exist. Anything else is placing constraint (or rather dependency) on the actual packaging and deployment. Sensible deployment should decide on the actual location of state file and there is a couple of ways to configure it; moreover in my opinion placing state file in the same directory as the binary installation is not a good practice.

Reply:
Ok, I'll leave it to @martin-kuba, since he knows deployment details.

Reply:
Docker containers keep their image between restarts, so the file /tmp/auditlogger.state would survive restart. But it will not survive deployment of a new version of the application. Is it a problem? We have the same situation with perun_ldapc, where the setting is `ldap.stateFile=./perun-ldapc-last-state` and the working directory is _outside_ of the container. So if you want to keep the state between different versions of the application, set it to `./auditerlogger.state`.

Reply:
OK, changed the default.